### PR TITLE
paket 1.14.0 (new formula)

### PR DESCRIPTION
**Pull request by simonhdickson**

### CODE REVIEW

File: Library/Formula/paket.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Paket < Formula
+  desc "Package dependency manager for .NET"
+  homepage "https://fsprojects.github.io/Paket/"
+  version "1.14.0"
+  url "https://github.com/fsprojects/Paket/archive/#{version}.tar.gz"
```
Comment (DomT4) [inline Library/Formula/paket.rb]: We don't do things this way; The Cask does, but Homebrew doesn't. Should be inlined into the `url`, i.e. `"https://github.com/fsprojects/Paket/archive/1.14.0.tar.gz"`

File: Library/Formula/paket.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      exec mono #{bin}/paket.exe "\$@"
+    EOS
+  end
+
+  test do
+    system "paket init"
+    assert (testpath/"paket.dependencies").exist?, "paket.dependencies file should be created"
+  end
+
+  def caveats; <<-EOS.undent
```
Comment (DomT4) [inline Library/Formula/paket.rb]: Scrap the caveats. Checking a project's homepage for answers should be fairly apparent to everyone.

**Comment by mikemcquaid** [comment]
Lots of stuff failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/27942/

**Comment by DomT4** [comment]
This needs rebasing on an up-to-date master and some `brew audit --strict` fixes. Thanks!

**Comment by simonhdickson** [comment]
I haven't been able to make it build using the fsharp brew formula (it currently requires you have fsharp installed outside of homebrew) and I haven't been able to/don't have the time to fix it properly. Am I better off closing this until its fixed and making the formula a tap?

**Comment by mikemcquaid** [comment]
Until it works with Homebrew's `fsharp` a tap is probably better, yeh. Sorry!

**Comment by srid** [comment]
Hey @simoncozens - did you end up hacking on anything else on top of this (such as creating a new tap)?

**Comment by simoncozens** [comment]
Don't think you mean me...

**Pull request closed without merging**